src/components/SignUp/ActorInfo2.tsx

```diff
@@ -199,7 +199,7 @@ const ActorInfo2: React.FC<{
               <Form.Group>
                 <CAGLabel>
```

Comment:
I think there's still a bit of confusion here. I think all we need to do is, in ActorInfo2, create a new styled component to replace these two CAGLabels, like so:

```
const CAGLabelSmaller = styled(CAGabel)`
  font-size: 16px;
`;
```

And then you can do:
```
<CAGLabelSmaller>
  I would also be comfortable playing roles usually played by:{' '}
  <PrivateLabel />
</CAGLabelSmaller>
```
And:
```
<CAGLabelSmaller>
  I would be comfortable playing a character through all phases
  of their transition: <PrivateLabel />
</CAGLabelSmaller>
```

Reply:
I'm so sorry!! Makes sense now; I don't know why I kept thinking by label you were referring to the private label and not the column text CAGLabel itself. Updated to reflect these changes.